mongoCluster/Driver.cs

```diff
@@ -239,18 +312,15 @@ private bool _collectionExists(String collectionName)
         /// <returns>True if successfully accessed, False, otherwise</returns>
         private bool _getCollection(String collectionName)
         {
-            if (!this._collectionExists(collectionName))
```

Comment:
I understood why this check might be taken out if we want the default behavior of a collection being created when calling `getCollection`. I added this to prevent new collections from being accidentally created on the clusters. For example, if somebody accidentally typed "listingss", it would prevent the incorrect collection from being created (and avoid unnecessary clean-up) considering the query scope and wanted behavior in this project. 

I'm curious what the reasoning was behind taking out the check case?